docs/CORTX-S3 Server Quick Start Guide.md

```diff
@@ -156,14 +156,14 @@ The image below illustrates the output log of a system test that is successful.
 
 Before your test your build, ensure that you have installed and configured the following:
 
-1. Make sure you have installed `easy_install`
-    * To check if you have `easy_install`, run the command: `$ easy_install --version`
-    * To install `easy_install`, run the command: `$ yum install python-setuptools python-setuptools-devel`
-2. Ensure you've installed `pip`
-    * To check if you have pip installed, run the command: `$ pip --version`
-    * To install pip, run the command: `$ python --version`
-3. If you don't have Python Version 2.6.5+, then install Python using: `$ python3 --version`
-    *  If you don't have Python Version 3.3, then install python3 using: `$ easy_install pip`
+1. Make sure you have installed easy_install.
+    - To check if you have easy_install, run the command: `$ easy_install --version`
+    - To install easy_install, run the command: `$ yum install python-setuptools python-setuptools-devel`

```

Comment:
we can remove back-ticks.

Reply:
do same for rest of the cmds in this doc.

Reply:
This is used for formatting the code in .md file @amitwac2608